lighthouse-core/audits/seo/font-size.js

```diff
@@ -157,12 +157,19 @@ function findStyleRuleSource(baseURL, styleDeclaration, node) {
         source = `${url.href}`;
 
         if (range) {
-          // `stylesheet` can be either an external file (stylesheet.startLine will always be 0),
-          // or a <style> block (stylesheet.startLine will vary)
-          const absoluteStartLine = range.startLine + stylesheet.startLine + 1;
-          const absoluteStartColumn = range.startColumn + stylesheet.startColumn + 1;
-
-          source += `:${absoluteStartLine}:${absoluteStartColumn}`;
+          // Add the startLine/startColumn of the <style> element to the range, if stylesheet is inline.
+          // Just use the rule's location if a sourceURL magic comment is present (`hasSourceURL` is true).
+          const addHtmlLocationOffset = stylesheet.isInline && !stylesheet.hasSourceURL;
+
+          const line = addHtmlLocationOffset ?
```

Comment:
i'd separate these concerns a little more but its no big deal really. calling out the +1 for lines but not columns is potentially worthwhile.

```js
          let line = range.startLine;
          line++; // lines are 0-indexed in the protocol, but users expect 1-indexed line values;
          if (addHtmlLocationOffset) line += stylesheet.startLine;
```

Reply:
ah nvm brendan's is nicer.